src/components/ErrorMessage/ErrorMessage.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const ErrorMessage = props => {
+	return <div style={{color: 'red'}}>Error: {props.text}</div>;
```

Comment:
We could create separate style sheets for components, e.g. `ErrorMessage.css` and follow bem:

```
// ErrorMessage.js
<p className="error-message">...</p>

// ErrorMessage.css

.error-message {
  color: red;
}
```